resources/views/layouts/app.blade.php

```diff
@@ -47,7 +47,7 @@
                                 aria-label="{{ __('notifications.mark-all-read') }}">
```

Comment:
<img width="1320" alt="Screenshot 2021-04-17 at 11 06 21" src="https://user-images.githubusercontent.com/4103693/115107719-f293f800-9f6c-11eb-9562-c68fdb44b821.png">
<img width="1174" alt="Screenshot 2021-04-17 at 11 06 58" src="https://user-images.githubusercontent.com/4103693/115107736-09d2e580-9f6d-11eb-90eb-8b00e7ea4b20.png">
The new container is wider than the old one. I personally do not like that. Can we manually make it a little smaller on the desktop? The layout and especially the status cards seem so long now.